app/jobs/cron/export_and_publish_demarches_publiques_job.rb

```diff
@@ -0,0 +1,18 @@
+class Cron::ExportAndPublishDemarchesPubliquesJob < Cron::CronJob
+  self.schedule_expression = "every month at 3:00"
+
+  def perform(*args)
+    gzip_filepath = [
+      ENV.fetch('DATAGOUV_TMP_DIR', 'tmp'), '/',
+      Time.zone.now.to_formatted_s(:number),
+      '-demarches.json.gz'
+    ].join
```

Comment:
possible de desactiver la feature pour les autres instances?
```suggestion
    return if ENV['DATAGOUV_TMP_DIR'].nil?
    gzip_filepath = [
      ENV.fetch('DATAGOUV_TMP_DIR'), '/',
      Time.zone.now.to_formatted_s(:number),
      '-demarches.json.gz'
    ].join
```

Reply:
Pour les autres instances, il faut qu'on trouve un moyen de ne pas enquer certains cron jobs. Nous en avons parlé avec @kara22. On pourrai déjà commencer par mettre les job qui concernent le datagouv dans un namespace à part pour clarifier. par exemple `Cron::DataGouv::ExportAndPublishDemarchesPubliquesJob`. T'en pense quoi @krichtof ?

Reply:
Oui du coup je pense que @krichtof il faudrait qu'on merge ta PR en premier comme cela je rebase et j'utilise ton code au lieu de le doubler.

Par contre comme vu avec Data Gouv, il serait bon qu'on utilise qu'un seul DataSet et j'en ai créé un avec 8 ressources, est-ce que tu pourrais utiliser celui la stp ?

Reply:
@tchak oui, d'accord avec toi d'avoir un même namespace.
@kara22 ok pour merger d'abord ma PR. Et pour mutualiser l'appel à l'API datagouv

J'ai vu avec Julieta de datagouv. Elle trouve pertinent d'avoir un dataset dédié aux stats, et un autre pour le descriptif des démarches.